### PR TITLE
fix: enrichCDXSupplier comma-separated websites

### LINKED ISSUE #84: Ecosyste.ms API returns comma-separated URLs in `repo_metadata.owner_record.website`
Given the title, `parlay` currently copies the `website` field as a single URL in the output CycloneDX, causing validation errors (e.g. in Dependency-Track).

See as example returned data for the package `pkg:golang/github.com/mailru/easyjson@v0.7.7` (https://packages.ecosyste.ms/api/v1/packages/lookup?purl=pkg%3Agolang%2Fgithub.com%2Fmailru%2Feasyjson%40v0.7.7):

```bash
curl 'https://packages.ecosyste.ms/api/v1/packages/lookup?purl=pkg%3Agolang%2Fgithub.com%2Fmailru%2Feasyjson%40v0.7.7' \
  | jq '.[].repo_metadata.owner_record.website'
```

Maybe `parlay` should try to split them on comma?
https://github.com/snyk/parlay/blob/327a30a795a858efd18ae262f465d77ade7f6d3f/lib/ecosystems/enrich_cyclonedx.go#L162-L165

### PR DESCRIPTION
**Pull request by acidghost**
Fixes #84.

### CODE REVIEW

File: lib/ecosystems/enrich_cyclonedx.go [hunk 1]
```diff
@@ -160,8 +161,11 @@ func enrichCDXSupplier(comp *cdx.Component, data *packages.Package) {
 					Name: name,
 				}
 				if website, ok := ownerRecord["website"].(string); ok {
-					websites := []string{website}
-					supplier.URL = &websites
+					split := strings.Split(website, ",")
```
Comment (mcombuechen) [inline lib/ecosystems/enrich_cyclonedx.go:164]:
```suggestion
					split := strings.Split(website, ", ")
```
I'm thinking that splitting on `","` can potentially break a URL with unescaped commas, e.g. `http://host/?foo=bar,baz`. Given ecosyste.ms's behaviour, splitting on `, ` seems safer.

**Review by mcombuechen — COMMENTED** [review]
Thanks for the PR @acidghost 

Please see my comment.

**Review by mcombuechen — APPROVED** [review]

**Comment by acidghost** [comment]
@mcombuechen The linting error is in code I haven't touched. I added 5f03d7ba82612f5e956c7f9b242d7272ef597d0a to fix it.

**Comment by acidghost** [comment]
> Thanks @acidghost looks good to me. Can you sign the commit?

@mcombuechen  Done

**Pull request merged**